samples/dnn/dnn_model_runner/dnn_conversion/paddlepaddle/README.md

```diff
@@ -20,6 +22,57 @@ python paddle_resnet50.py
 
 there are 3 part of this execution
 
-- 1. Export PaddlePaddle ResNet50 model to onnx format;
-- 2. Use `cv2.dnn.readNetFromONNX` load model file;
-- 3. Preprocess image file and do inference.
+* 1. Export PaddlePaddle ResNet50 model to onnx format;
+* 2. Use `cv2.dnn.readNetFromONNX` load model file;
+* 3. Preprocess image file and do inference.
+
+## 2. Run PaddleSeg Portrait Segmentation by OpenCV
+
+### Convert to ONNX Model
+
+#### Step1. get Paddle Inference model
+
+Thanks for [PaddleSeg](https://github.com/PaddlePaddle/PaddleSeg), this Paddle Inference model convert from HRNet w18 small v1. For more details, please refer to [HumanSeg](https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.1/contrib/HumanSeg/README.md).
+
+```shell
+wget https://x2paddle.bj.bcebos.com/inference/models/humanseg_hrnet18_small_v1.zip
+unzip humanseg_hrnet18_small_v1.zip
+```
+
+Notes:
+
+* The exported Paddle Inference model has to be fixed input shape while dynamic shape is not supported by opencv now.
+
+#### Step2. convert to ONNX model by paddle2onnx
+
+Use the following command to convert
+
+```
+paddle2onnx --model_dir humanseg_hrnet18_small_v1 \
+            --model_filename model.pdmodel \
+            --params_filename model.pdiparams \
+            --opset_version 11 \
+            --save_file humanseg_hrnet18_tiny.onnx
+```
+
+The converted model is the one under the current path `humanseg_hrnet18_tiny.onnx` .
+
+### Run PaddleSeg Portrait Segmentation demo
+
+Run the example code as below,
+
+```shell
+python paddle_humanseg.py
+```
+
+there are 3 part of this execution
+
+* 1. Use `cv2.dnn.readNetFromONNX` load model file;
+* 2. Preprocess image file and do inference.
+* 3. Postprocess image file and visualization.
+
+The result is in the path `data/result_test_human.png` .
```

Comment:
.png files should be avoided in documentation (it is lossless format which provides huge files).
Consider using .jpg image format in documentation.

Reply:
What is about the source of used image?

Reply:
The source of used image comes from Supervise.ly dataset,it is noted in the last line of the readme~

Reply:
Used images should have proper licensing terms. Do you have a link on them?

Reply:
The sample image comes from [supervisely-ecosystem/persons](https://github.com/supervisely-ecosystem/persons#download),it's publicly available and free for academic purposes, and there's no license announcement for the dataset. Do you have any suggestions~